app/src/main/res/values/preference_keys.xml

```diff
@@ -257,7 +254,8 @@
     <string name="pref_key_show_collections_placeholder_home" translatable="false">pref_key_show_collections_home</string>
 
     <!-- Tabs Settings -->
-    <string name="pref_key_tab_view_list" translatable="false">pref_key_tab_view_list</string>
+    <!-- pref_key_tab_view_list_do_not_use is needed for UI implementation only and should be expected to be useful. -->
+    <string name="pref_key_tab_view_list_do_not_use" translatable="false">pref_key_tab_view_list</string>
```

Comment:
```suggestion
    <string name="pref_key_tab_view_list_do_not_use" translatable="false">pref_key_tab_view_list_do_not_use</string>
```

Reply:
We need this value to stay the same so that we don't break existing users. We don't want to reset their prefs if they explicitly set it to list.